zkSNACKs/WalletWasabi
# Fetch transaction effective feeRate from backend

Alternative to #11613

Needed for https://github.com/zkSNACKs/WalletWasabi/issues/12147

Tasks:

- [x] Figure out acceptable UX (loading wheel?)
- [x] Delay requests
- [x] new Tor circuit for every request?
- [x] Detect and refresh Fee properties when Fee arrives from backend

**How to test this on RegTest:**

1. Run Knots, backend, client
2. Receive a payment from outside of the Wasabi Client (Bitcoin Knots)
3. Open the details of the incoming (pending) transaction.
4. See that we were able to show the feeRate and the estimated confirmation time, even if the transaction contains `Foreign` inputs.
